src/App.css

```diff
@@ -1,14 +1,78 @@
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:wght@300;400;500;600&family=Saira:wght@300;400;500;600;700&display=swap');
+
+:root {
+  --font-size-m: 16px;
+  --font-size-s: 12px;
+  --space-xxl: 40px;
+  --space-xl: 28px;
+  --space-l: 20px;
+  --space-m: 14px;
+  --space-s: 10px;
+  --space-xs: 6px;
+  --space-xxs: 3px;
+  --border-radius-s: 4px;
+  --red: red;
+  --red-light: #fbbebe;
+  --blue-darkest: #2d374b;
+  --blue-dark: #1683cd;
+  --blue: #28a0f0;
+  --blue-lighter: #96bedc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: var(--font-size-m);
+  background-image: url('https://bridge.arbitrum.io/images/space.webp');
+  background-repeat: repeat;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  font-family: 'DM Sans', -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background: linear-gradient(179.53deg,#000 19.46%,rgba(0,0,0,.650795) 40.76%,rgba(0,0,0,.462631) 60.09%,hsla(0,0%,100%,0) 99.6%);
+  padding: var(--space-xxl) var(--space-m);
+  margin: 0;
+}
+
+h1 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: var(--space-s);
+  font-family: 'Saira', sans-serif;
+  font-size: 2.8rem;
+  line-height: 1.3;
+  margin-top: 0;
+}
+
+@media screen and (max-width: 400px) {
+  h1 {
+    display: inline;
+  }
+}
+
 .form-container {
-  margin-top: 20px;
   display: flex; /* or inline-flex */
   flex-direction: row;
   flex-wrap: nowrap;
+  width: 100%;
+  max-width: 740px;
   gap: 10px;
 }
 
 .input-style {
-  font-size: 16px;
-  line-height: 28px;
+  font-size: var(--font-size-m);
+  font-family: 'Lato', sans-serif;
```

Comment:
I think we could reuse either `Saira` or `DM Sans` here (or load Lato, as it's not loaded currently)